src/main/java/org/corfudb/infrastructure/LayoutServer.java

```diff
@@ -289,12 +285,15 @@ public synchronized  void handleMessageLayoutRequest(CorfuMsg msg, ChannelHandle
         if (msg.getEpoch() <= serverContext.getServerEpoch()) {
```

Comment:
To be consistent, we should add epoch to the payload of LayoutRequest messages, and check from the payload, rather than msg.getEpoch